sampradAyaH/kaulaH_shAktaH/sampradAyaH/daxiNAmnAyaH_shrI-kulam/subrahmaNyAryaH_chidAnandanAthaH/shrI-vidyA-saparyA-paddhatiH.md

```diff
@@ -9,57 +9,54 @@ TODO: परिष्कार्यम्
 
 ་ 
 f 
-ओ श्रीगुरुभ्यो नमः । 
-.ओ श्रीमहागणपतये नमः । 
-ओं श्रीललितामहात्रिपुरसुन्दर्यै नमः । 
+ओं श्रीगुरुभ्यो नमः । 
```

Comment:
https://sanskrit.github.io/groups/dyuganga/projects/text/proofreading/editing/markdown/Spacing_paragraph-vivekaH/ इत्य् अनुसरतु। 
विशिष्यात्र "Of course, some paragraphs have multiple lines (example - halves of a shloka). In such cases, there should be TWO SPACES at the end of each line." इति प्रसज्यते।

Reply:
आम् इदानीं ज्ञातवती।

Reply:
श्लोकस्य कृते two spaces स्थापितवती। Paragraph ज्ञातुं Enter कृत्वा 1 पङ्क्ती त्यक्तवती।

Reply:
"ओं श्रीगुरुभ्यो नमः । " इत्यस्मात् परं त्व् एक एवावकाशो दृश्यते खलु?

Reply:
"ओं नमो ब्रह्मादिभ्यो ब्रह्मविद्यासंप्रदायकर्तृभ्यो वंशर्षिभ्यो नमो गुरुभ्यः । सर्वोपप्लवरहितप्रज्ञानघनप्रत्यगर्थो ब्रह्माहमस्मि,सोहमस्मि, ब्रह्माहमस्मि ॥" इत्यस्मात् परम् अपि रिक्तस्थानद्वयाभावः  - परीक्षताम्।

Reply:
> रिक्तस्थानद्वयं स्थापितवती।